### PR TITLE
Studio deploys 2024-09-05

### PR DESCRIPTION
**Pull request by lindsgil**
## Description of the change

Studio Deployments 09-05-2024

Checklist:
- [x] review
- [x] queue deployments
- [x] execute deployments
- [x] run post deployment scripts

### CODE REVIEW

File: packages/contracts/deployments/engine/V3/studio/mainnet/2024-09-05-deployment-config.ts [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+  // ONLY Gnosis Safe is supported for Studio deployments at this time
+  useGnosisSafe: true,
+  safeAddress: "0x52119BB73Ac8bdbE59aF0EEdFd4E4Ee6887Ed2EA",
+  transactionServiceUrl: "https://safe-transaction-mainnet.safe.global",
+  // set the transaction hash after successful execution of the batch creation
+  transactionHash: "",
+};
+
+export const deployConfigDetailsArray = [
+  {
```
Comment (ryley-o) [inline packages/contracts/deployments/engine/V3/studio/mainnet/2024-09-05-deployment-config.ts:19]: please add `productClass: "Studio"` to these (see deployment-config.template.ts)

File: packages/contracts/deployments/engine/V3/studio/mainnet/2024-09-05-deployment-config.ts [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+    // common values include `studio`, `fullyonchain`, `flex`, or partnerships like `artblocksxpace`.
+    // also note that if you desire to create a new vertical, you will need to add the vertical name to the
+    // `project_verticals` table in the database before running this deploy script.
+    defaultVerticalName: "studio",
+  },
+  {
+    // 0 for Engine, 1 for Engine Flex
+    engineCoreContractType: 1,
+    // prod-only: see efficient_addresses...txt in network directories for a list of efficient salts
+    salt: "0x0000000000000000000000000000000000000000a73bf3db9deb5f1cfa0a0080",
```
Comment (ryley-o) [inline packages/contracts/deployments/engine/V3/studio/mainnet/2024-09-05-deployment-config.ts:124]: this needs to be removed from the flex efficient addresses file 😃

**Review by ryley-o — APPROVED** [review]
LGTM - minor updates from a couple changes I made for finance recently

File: packages/contracts/deployments/engine/V3/studio/mainnet/2024-09-05-deployment-config.ts (hunk 1 above)
Comment (lindsgil) [reply packages/contracts/deployments/engine/V3/studio/mainnet/2024-09-05-deployment-config.ts:19]: 🫡

File: packages/contracts/deployments/engine/V3/studio/mainnet/2024-09-05-deployment-config.ts (hunk 2 above)
Comment (lindsgil) [reply packages/contracts/deployments/engine/V3/studio/mainnet/2024-09-05-deployment-config.ts:124]: thank you !!!

**Pull request merged**